be/src/storage/persistent_index.cpp

```diff
@@ -497,7 +497,7 @@ Status ImmutableIndexWriter::write_shard(size_t key_size, size_t npage_hint, siz
         _total_kv_size += (key_size + kIndexValueSize) * kvs.size();
     } else {
         _total_kv_size +=
-                std::accumulate(kvs.begin(), kvs.end(), 0, [](size_t s, const auto& e) { return s + e.size; });
+                std::accumulate(kvs.begin(), kvs.end(), (size_t)0, [](size_t s, const auto& e) { return s + e.size; });
```

Comment:
Won't it be automatically converted to size_t here?

Reply:
> Won't it be automatically converted to size_t here?

No auto conversion from test results